doc/source/cookbook.rst

```diff
@@ -776,11 +776,14 @@ Resampling
 
 The :ref:`Resample <timeseries.resampling>` docs.
 
-`TimeGrouping of values grouped across time
-<http://stackoverflow.com/questions/15297053/how-can-i-divide-single-values-of-a-dataframe-by-monthly-averages>`__
```

Comment:
@jorisvandenbossche thank you for reviewing. I'm sorry I'm new to this, it'll be great if you could guide me on this.

I'll definitely try and update the stackoverflow post. 

> In any case, the Grouper docs in itself are not a good reference for "Using Grouper to group values across time"

Should I instead include some examples to show the usage of Grouper to group values across time?